response_operations_ui/templates/collection_exercise/ce-eq-instrument-section.html

```diff
@@ -19,16 +19,24 @@
     {% include 'collection_exercise/ce-success-panel.html' %}
     {% include 'collection_exercise/ce-info-panel.html' %}
     {% include 'partials/flashed-messages.html' %}
-    <h1 name="page-survey-title" class="ons-u-mb-s">EQ collection instruments | {{ survey.shortName }} {{ survey.surveyRef }}</h1>
-
+    <h1 class="ons-u-fs-m ons-u-lighter" name="page-survey-subtitle">{{ survey.longName }} | {{ ce.exerciseRef }}</h1>
+    <h1 name="page-survey-title" class="ons-u-mb-s">Collection instruments step 1 of 2</h1>
+    {% if cir_enabled %}
+        {% if not locked and surveyEditPermission %}
+            <p>
```

Comment:
Moving this outside of the grids to improve the spacing between the heading and content